Container/Container.cs

```diff
@@ -1,57 +1,33 @@
-﻿using System.ComponentModel.DataAnnotations;
-
-namespace Container
+﻿namespace Container
 {
     public class Container
     {
-        List<int> container = new List<int>();
-        public int maxLength = 5;
+        private readonly List<int> _container = new(capacity: 5);
+
+        public int MaxLength { get; } = 5;
```

Comment:
To rzeczywiście może być publiczne property, bo jest to dosyć istotna informacja dla klientów tej klasy. Najlepiej wtedy użyć właśnie property, a nie field, bo możemy np. określić, że zezwalamy jedynie na odczyt tej danej (`get`). Stosując publiczne pole nie możemy dodać takiego obostrzenia. Generalnie zawzse kiedy masz jakąś publiczną daną w klasie, będzie to property, a nie field.